### PR TITLE
Make it possible to change use_slots online

### PR DESCRIPTION
**Pull request by CyberDem0n**
Previously it required restarting Patroni and removing slots manually.

Fixes https://github.com/zalando/patroni/issues/1158

### CODE REVIEW

File: patroni/dcs/__init__.py [hunk 1]
```diff
@@ -442,14 +442,16 @@ def get_replication_slots(self, name, role):
         # the current master, because that member would replicate from elsewhere. We still create the slot if
         # the replicatefrom destination member is currently not a member of the cluster (fallback to the
         # master), or if replicatefrom destination member happens to be the current master
+        use_slots = self.config and self.config.data.get('postgresql', {}).get('use_slots', True)
```
[the remark this answers is not in the document]
Comment (CyberDem0n) [reply patroni/dcs/__init__.py]: It is in a different PR: https://github.com/zalando/patroni/pull/1244/files#diff-40772a77bd7d4ea1f3cb4cbef17a813dR23

**Comment by CyberDem0n** [comment]
> 2019-10-31 10:56:09,105 ERROR: Failed to drop replication slot 'postgresql0'

The slot is still in use. That happens because replicas didn't updated the recovery.conf and restarted yet. Master will continue attempts until it succeeded.

**Comment by Jan-M** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Comment by dtseiler** [comment]
Nice, thanks!

**Pull request merged**